ome/provider.go

```diff
@@ -219,6 +219,7 @@ func (p *omeProvider) DataSources(ctx context.Context) []func() datasource.DataS
 		NewVlanNetworkDataSource,
```

Comment:
@@ -219,6 +219,7 @@ func (p *omeProvider) DataSources(ctx context.Context) []func() datasource.DataS
 		NewVlanNetworkDataSource,
 		NewConfigurationReportDataSource,
 		NewDeviceDatasource,
+		NewAppCertDataSource,
 	}
 }

Action:
I would recommend adding a comment or a note to explain the purpose of the new data source, `NewAppCertDataSource`. This is good practice for maintainability and understanding the code.

Additionally, it would be beneficial to group the data sources in a meaningful way, such as by category or functionality. This would improve code readability and make it easier to understand the relationships between the data sources.

No other issues were identified in the given diff. The new data source appears to be properly added, but it might benefit from additional context and organization.